keyboards/xd68/xd68.c

```diff
@@ -0,0 +1,31 @@
+#include "xd68.h"
+#include "led.h"
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+	matrix_init_user();
+	led_init_ports();
+};
+
+void matrix_scan_kb(void) {
+	// put your looping keyboard code here
+	// runs every cycle (a lot)
+	matrix_scan_user();
+};
+
+void led_init_ports(void) {
+    // * Set our LED pins as output
+    DDRB |= (1<<2);
+}
+
+void led_set_kb(uint8_t usb_led) {
+	if (usb_led & (1<<USB_LED_CAPS_LOCK)) {
+        // Turn capslock on
+        PORTB &= ~(1<<2);
+    } else {
+        // Turn capslock off
+        PORTB |= (1<<2);
+    }
+	led_set_user(usb_led);
```

Comment:
This function should be changed to:

```c
bool led_update_kb(led_t led_state) {
    if(led_update_user(led_state)) {
        writePin(B2, !led_state.caps_lock);
    }
    return true;
}
```

This will allow user keymaps to override this for things like layer indication.